app.rb

```diff
@@ -0,0 +1,96 @@
+require './person'
+require './student'
+require './teacher'
+require './classroom'
+require './corrector'
+require './book'
+require './rental'
+
+class App
+  attr_reader :books, :people, :rentals
+
+  def initialize(books = [], people = [], rentals = [])
+    @books = books
+    @people = people
+    @rentals = rentals
+  end
+
+  def all_books
+    @books.each { |book| puts "Title: \"#{book.title}\", Author: #{book.author}" }
+  end
+
+  def all_people
+    @people.each { |person| puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}" }
+  end
+
+  # rubocop:disable Metrics/MethodLength
+  def create_person
+    print 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+    option_person = gets.chomp
+    print 'Age: '
+    age = gets.chomp
+    age = 18 if age == ''
+    print 'Name: '
+    name = gets.chomp
+    name = 'Unknown' if name == ''
+
+    case option_person
+    when '1'
+      print 'Has parent permission? [Y/N]: '
+      has_parent_permission = gets.chomp
+      has_permission = has_parent_permission.downcase == 'y'
+      person = Student.new(age: age, name: name, parent_permission: has_permission)
+
+    when '2'
+      print 'Specialization: '
+      specialization = gets.chomp
+      specialization = 'Unknown' if specialization == ''
+      person = Teacher.new(age: age, specialization: specialization, name: name)
+    else
+      puts 'Sorry, you choose a wrong option'
+      return
+    end
+    @people << person unless @people.include?(person)
+    puts 'Person created successfully'
+  end
+  # rubocop:enable Metrics/MethodLength
```

Comment:
- [ ]  Kindly remove this and use `rubocop.yml` to achieve this purpose in order to make your file clean. More so, commented-out code makes our work look messy. ℹ️